include/Core/UI/Setting/EmailLogSetting.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace EmailLog\Core\UI\Setting;
+use EmailLog\Core\UI\Setting\SettingField;
+
+class EmailLogSetting extends Setting {
+
+	/**
+	 * Implement `initialize()` method.
+	 */
+	protected function initialize() {
+		$this->section->id          = 'email-log';
+		$this->section->title       = __( 'Email Log Settings', 'email-log' );
+		$this->section->option_name = 'el_email_log';
```

Comment:
Let's change this to `el_email_log_core`

Reply:
Fixed.